[2024-04-05T16:07:38.692+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this scheduled__2024-04-03T00:00:00+00:00 [queued]>
[2024-04-05T16:07:39.467+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this scheduled__2024-04-03T00:00:00+00:00 [queued]>
[2024-04-05T16:07:39.474+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-04-05T16:07:39.507+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-04-05T16:07:39.511+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-04-05T16:07:40.945+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): also_run_this> on 2024-04-03 00:00:00+00:00
[2024-04-05T16:07:41.129+0000] {standard_task_runner.py:55} INFO - Started process 3182 to run task
[2024-04-05T16:07:41.379+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'also_run_this', 'scheduled__2024-04-03T00:00:00+00:00', '--job-id', '97', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmp_rqef2ml']
[2024-04-05T16:07:41.458+0000] {standard_task_runner.py:83} INFO - Job 97: Subtask also_run_this
[2024-04-05T16:08:08.258+0000] {local_task_job.py:221} WARNING - DagRun timed out after 1 day, 12:16:03.258316.
[2024-04-05T16:08:17.832+0000] {local_task_job.py:221} WARNING - DagRun timed out after 1 day, 12:16:13.003412.
[2024-04-05T16:08:18.111+0000] {local_task_job.py:228} WARNING - State of this instance has been externally set to skipped. Terminating instance.
[2024-04-05T16:08:18.526+0000] {process_utils.py:133} INFO - Sending Signals.SIGTERM to group 3182. PIDs of all processes in the group: [3182]
[2024-04-05T16:08:18.654+0000] {process_utils.py:84} INFO - Sending the signal Signals.SIGTERM to group 3182
[2024-04-05T16:08:20.531+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=3182, status='terminated', exitcode=<Negsignal.SIGTERM: -15>, started='16:07:40') (3182) terminated with exit code Negsignal.SIGTERM
